pymongo/asynchronous/cursor.py

```diff
@@ -294,6 +295,7 @@ def _clone(self, deepcopy: bool = True, base: Optional[AsyncCursor] = None) -> A
             "limit",
             "max_time_ms",
             "max_await_time_ms",
+            "timeout",
```

Comment:
We shouldn't need to add timeout here since it's already set implicitly in _clone_base.

Reply:
hm i didn't at first but i was failing `test_clone` and adding this seemed to fix it >.< 
i'll undo this change in my next commit though in case i'm mis-remembering